src/test/java/com/snowflake/kafka/connector/internal/SnowflakeIngestionServiceV1Test.java

```diff
@@ -0,0 +1,122 @@
+package com.snowflake.kafka.connector.internal;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicReference;
+import net.snowflake.ingest.SimpleIngestManager;
+import net.snowflake.ingest.connection.HistoryResponse;
+import net.snowflake.ingest.connection.IngestStatus;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+class SnowflakeIngestionServiceV1Test {
+
+  private SimpleIngestManager manager;
+  private SnowflakeIngestionService svc;
+
+  @BeforeEach
+  void setup() {
+    manager = Mockito.mock(SimpleIngestManager.class);
```

Comment:
Ideally we have our own fake (and abstraction over it SimpleIngestManager) instead of yet another mock.

Reply:
maybe, but there were none so far and I wanted to provide some basic testing for the service itself, without relying on the underlying SDK class